src/csharp/Grpc.Core.Api/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,63 @@
+#region Copyright notice and license
+
+// Copyright 2018 The gRPC Authors
```

Comment:
nit: 2019? Also, I noticed that we don't generally add copyright to our AssemblyInfo.cs files, but if it's expected in grpc/grpc-dotnet I'll update the PR https://github.com/grpc/grpc-dotnet/pull/48

Reply:
will update to 2019.

We usually add copyright notice to all source files (and in this particular case I think grpc/grpc has a license checker test that requires that the .cs files have the license notice in place.).

Reply:
I will update my PR as well then. If there's a license checker test, I'll also add the license header to our examples. Generally we don't license the examples in our repos but we'll follow the grpc standards.

Reply:
to clarify: there's a license checker test setup on grpc/grpc, on grpc-dotnet we currently only have the travis tests. But yes, adding a license notice to all the files won't harm.